app/assets/stylesheets/application.scss

```diff
@@ -16,6 +16,30 @@
 
  */
 
- .tattoo_image {
-   width: 150px;
- }
+body {
+  background: url(/background.jpg) no-repeat center center fixed;
+  -webkit-background-size: cover;
+  -moz-background-size: cover;
+  -o-background-size: cover;
+  background-size: cover;
+}
+
+
+.tattoo_image {
+  max-height: 450px;
+  max-width: 500px;
+}
+
+.side_bar {
+  background-color: #333333;
```

Comment:
Color literals like `#333333` should only be used in variable declarations; they should be referred to via variable everywhere else.